daphne_service_utils/src/config.rs

```diff
@@ -0,0 +1,81 @@
+use daphne::{
+    hpke::{HpkeConfig, HpkeReceiverConfig},
+    DapGlobalConfig, DapVersion,
+};
+use serde::{Deserialize, Serialize};
+use url::Url;
+
+use crate::{auth::DaphneWorkerAuthMethod, DapRole};
+
+/// draft-wang-ppm-dap-taskprov: Long-lived parameters for the taskprov extension.
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct TaskprovConfig {
+    /// HPKE collector configuration for all taskprov tasks.
+    pub hpke_collector_config: HpkeConfig,
+
+    /// VDAF verify key init secret, used to generate the VDAF verification key for a taskprov task.
+    #[serde(with = "hex")]
+    pub vdaf_verify_key_init: [u8; 32],
+
+    /// Leader, Helper: Method for authorizing Leader requests.
+    pub leader_auth: DaphneWorkerAuthMethod,
+
+    /// Leader: Method for authorizing Collector requests.
+    pub collector_auth: Option<DaphneWorkerAuthMethod>,
+}
+
+pub type HpkeRecieverConfigList = Vec<HpkeReceiverConfig>;
+
+/// Daphne-Worker configuration, including long-lived parameters used across DAP tasks.
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct DaphneServiceConfig {
+    pub env: String,
```

Comment:
Missing documentation (not clear from the "some-machine-identifer" example what the intended semantics is).